contact_us.html

```diff
@@ -279,7 +279,7 @@ <h2 class="my-4 text-3xl font-bold tracking-tight text-green-600 sm:text-4xl">
         </form>
       </div>
       <div class="image-section lg:w-1/2 lg:py-16 lg:py-0 flex justify-between object-cover">
-        <img class="rounded-md bg-white/5 ring-1 ring-white/10" src="images/african_student.jpg" alt="App screenshot"
+        <img class="rounded-md bg-white/5 ring-1 ring-white/10" src="images/Child puzzles.png" alt="App screenshot"
```

Comment:
the name for the images should not have spaces in between. for example it should be 

child_puzzles.png